# Exception raised when the game is over
class GameOver(Exception):
    pass


# Exception raised when the enemy is defeated
class EnemyDown(Exception):
    pass
